### PR TITLE
Make row assertion in sample tests account for non-deterministic order

### PR DESCRIPTION
**Pull request by jonparrott**
This was causing some fun failures: https://circleci.com/gh/GoogleCloudPlatform/google-cloud-python/5213?utm_campaign=build-failed&utm_medium=email&utm_source=notification

### CODE REVIEW

File: docs/bigquery/snippets.py [hunk 1]
```diff
@@ -491,7 +491,9 @@ def do_something(row):
     ]
     assert len(rows) == total == 2
     assert token is None
-    assert rows == expected_rows
+    # Order is not preserved, so compare individually
+    for row in expected_rows:
+        assert row in rows
```
Comment (dhermes) [inline docs/bigquery/snippets.py]: You also need a length check?

Comment (jonparrott) [reply docs/bigquery/snippets.py]: The length check is above.

Comment (dhermes) [reply docs/bigquery/snippets.py]: Ah MERP! Since there is only two, it seems a for loop is "overkill"

Comment (jonparrott) [reply docs/bigquery/snippets.py]: Eh, yeah, still a useful assertion I think.

Comment (dhermes) [reply docs/bigquery/snippets.py]: The `for` loop isn't an assertion? I'm saying, just do:

```python
row1 = bigquery.Row(('Wylma Phlyntstone', 29), {'full_name': 0, 'age': 1})
assert row1 in rows
row2 = bigquery.Row(('Phred Phlyntstone', 32), {'full_name': 0, 'age': 1})
assert row2 in rows
```

Comment (jonparrott) [reply docs/bigquery/snippets.py]: Oh I see, easy enough. Done in e07a8620b.

**Pull request merged**